core/src/main/java/org/infinispan/manager/impl/ReplicableManagerFunctionCommand.java

```diff
@@ -61,6 +61,7 @@ public boolean isReturnValueExpected() {
 
    @Override
    public boolean canBlock() {
-      return true;
+      // Note that it is highly possible that a user command could block, and some internal Infinispan ones already do
+      return false;
```

Comment:
Since you created a separate ISPN-11483, we should return `true` here for now.

Reply:
The problem is we invoke some commands that invoke stores and it causes issues.

Reply:
Please add a comment to explain why you're returning false even though it *can* block.

Reply:
Sure I forgot to put the JIRA in the comments.